README.md

```diff
@@ -1,2 +1,168 @@
-# gutenberg-license
-Stores the grants of consent and tooling for Gutenberg's dual license
+# Readme
+
+This repository holds the information relevant to the requests for consent from [Gutenberg's](https://github.com/WordPress/gutenberg) contributors to dual-license their past contributions to Gutenberg under the GPL and MPL (more information available here). For more information, you can look at the [initial discussion issue on GitHub](https://github.com/WordPress/gutenberg/issues/23651) as well as the [initial proposal post](https://make.wordpress.org/core/2020/09/21/proposal-dual-licensing-gutenberg-under-gpl-v2-0-and-mpl-v2-0/) on 21 September 2020, and [updated proposal post](https://make.wordpress.org/core/2021/03/05/dual-licensing-gutenberg-next-steps/) on Make/Core.
```

Comment:
> more information available here

Is the "here" meant to be a link? Also, the next sentence starts "For more information" so it sounds like there's some redundancy there.

Reply:
I decided to expand that into a full sentence (which follows) and forgot to take out the parenthetical. I'll remove the parenthetical.

Reply:
Sounds good! I can't mark this as resolved so I'll leave that up to you.